free-1.md

```diff
@@ -4,40 +4,120 @@
+new, difficult reactions that haven't been seen before.  Thus they either have
+to use expensive software that eats up their grant money (and may not meet 
+their needs) or they have to take a long time to work it out by hand.  With a
+free tool (language) designed to be both expressive and intuitive, this process
+could be greatly improved (especially if the tool/language were designed to work
+alongside existing databases of reaction pathways).
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A reaction is generally predicted using well-specified rules that apply to 
+specific molecules and conditions.  This lends itself well to an imperative 
+style such as
+
+```
+Molecule hydronium = OH-
+Molecule hydroxide = H30+
+Molecule[] products = react(hydronium, hydroxide, conditions)
+products == [(Molecule) H20, (Molecule) H20]
+```
+
+ALternatively, when seeking synthesis pathways (i.e. a known endpoint) recursion 
+works well, such as
+
+```
+Molecule endproduct = <something complicated>
+Molecule startingpoint = synthesize(endproduct)
+```
+
+where `synthesize` is some sort of recursive function.
+
+It meshes well with existing programming language paradigms and designs.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+What excites me about this is that I've been working on something similar since 
+my sophomore year, when I was still a premed student taking Organic Chemistry.
+
+I thought to myself that these rules were relatively straightforward, and it was
+very much an algorithmic process.  I've been working on it off and on since 
+then.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Predicting organic chemistry reaction pathways
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+They could use an internal DSL; a library could be written using some preferred
+language and then provide hooks for some end-user to interact with.
+
+They could use an external DSL; a friendlier syntax that provides more 
+constraints on what can be done, and how.
+
+They could use a GUI-based external DSL; they draw molecules and conditions and
+then press a big "react" button.  This is probably the most end-user friendly,
+especially for chemists who aren't great with programming.  
+
+Overall I think each of these should be provided and well linked, if the DSL
+was well funded and supported.  This allows users of many technical backgrounds
+to interact with the software and customize it to meet their needs.  In context 
+to 111, the best is probably internal - this is the easiest to write and test
+because there are fewer layers between the algorithms and the user, and it 
+allows the most technical users to use it intimately and find the pain points.
```

Comment:
I agree with this, though if you feel like something more ambitious an external DSL could be cool too! I'd definitely recommend against an external DSL with a GUI for this class otherwise you'll probably spend almost all of your time making the GUI work (which I'd call part of the systems aspect of the project).